doc/project/checklist.md

```diff
@@ -111,6 +111,19 @@ FPGA synthesis timing meet (Fmax-10%) target or better
 CDC Sync flops use behavioral synchronization macros(`prim_flop_2sync`) not
 2flops.
 
+### SEC_CM_IMPLEMENTED
```

Comment:
This would in theory exclude things like AST-type countermeasures that we cannot talk about right?
There is some early discussion on making AST a comportable IP, and in those cases it wouldn't be documented so clearly.

Reply:
Technically this should only include items in the open source repo, so those would be special cases. But we haven't discussed the duality of that w.r.t. D1/D2/D3 etc, so let's just focus on the open source piece.